css/components/dashboards/dashboards-gallery.scss

```diff
@@ -0,0 +1,142 @@
+.c-dashboards-gallery-list {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  .thumbnail-list-item {
+    position: relative;
+    height: 200px;
+    border-radius: 4px;
+    background-color: $color-dark-grey;
+    background-size: cover;
+    background-position: center center;
+    outline: none;
+    padding: 0;
+    transition: all $animation-time-2 $ease-in-out-sine;
+
+    &:nth-of-type(6n) {
+      margin-right: 0;
+    }
+
+    // The input is just here for an accessibility matter
+    // we don't want it to be visually present
+    input[type="radio"] {
```

Comment:
I don't see any `input` included as part of the dashboard gallery, thus I don't understand this CSS rule definition here.

Reply:
Fixed here: https://github.com/resource-watch/resource-watch/pull/1185/commits/68daf5ece578aa9e34947bc6518d91f784da794a